### ISSUE TITLE
✨ Liveness detection

### ISSUE THREAD

**Issue by ShriN** [cuvent/react-native-vision-camera#3343]
### What feature or enhancement are you suggesting?

Can you please add a feature of liveness detection in captured photo or snapshot? We want to force users to upload real to capture their real face and not the face from picture.
Thanks for creating such a nice efficient library.

### What Platforms whould this feature/enhancement affect?

iOS, Android

### Alternatives/Workarounds

We are struct because of not having this option. Let me know if you have some workaround until this is included into the library. Thanks.

### Additional information

- [ ] I agree to follow this project's [Code of Conduct](https://github.com/mrousavy/react-native-vision-camera/blob/main/CODE_OF_CONDUCT.md)
- [X] I searched for [similar feature requests in this repository](https://github.com/mrousavy/react-native-vision-camera/issues?q=is%3Aopen+is%3Aissue+label%3A%22✨+enhancement%22) and found none.

**Comment by mrousavy** [comment]
No. This is something that would go in a Frame Processor or custom code. Not VisionCamera.
